app/Services/FacultyProfileService.php

```diff
@@ -18,11 +18,9 @@ public function __construct(WebResourceRetrieverContract $webResourceUtility)
 
     public function getFacultyProfile($email)
     {
-        $mediaURI = env('MEDIA_URL') . $email;
+        $emailUri = \substr($email, 0, \strpos($email, '@'));
 
-        // if (is_null($member['profile_image'])) {
-        //     $member['profile_image'] = asset('images/likeness.jpg');
-        // }
+        $mediaURI = env('MEDIA_URL') . 'faculty/media/' . $emailUri . '/avatar/?source=true';
```

Comment:
This still loads but it should follow the RESTful pattern, meaning: let's not have a slash before the question mark
```suggestion
        $mediaURI = env('MEDIA_URL') . 'faculty/media/' . $emailUri . '/avatar?source=true';
```